### ISSUE TITLE
TraceTelemetry w/Severity set is no longer appearing in Insights

### ISSUE THREAD

**Issue by FBNitro** [Microsoft/ApplicationInsights-Java#436]
I picked up Version 1.0.10 yesterday.  After doing so, all my trace lines with severities set stopped writing to application insights.

ie:
```
TraceTelemetry traceTelemetry = new TraceTelemetry("hello", SeverityLevel.Information);
 OperationContext operation = traceTelemetry.getContext().getOperation();
 operation.setId("MyTransactionId");
 telemetry.trackTrace(traceTelemetry);
```

The above stopped tracing with 1.0.10

while if I do the same thing, but set the SeverityLevel to null, it works:
```
TraceTelemetry traceTelemetry = new TraceTelemetry("hello", null);
 OperationContext operation = traceTelemetry.getContext().getOperation();
 operation.setId("MyTransactionId");
 telemetry.trackTrace(traceTelemetry);
```

I rolled back to 1.0.9 because of this defect.

**Comment by dhaval24** [comment]
Thanks for pointing this out.  In our preliminary investigation it seems that there is an issue with the TraceTelemetry class and we believe this is a critical issue and would be fixed as soon as possible in the next release.

**Comment by dhaval24** [comment]
@FBNitro there is a pull request coming in #437 which will fix this. If you would like to get this working before official release, as soon as it is merged in the master you can build the master locally and run the latest version. Thanks for reporting this issue.

**Comment by HeikoH** [comment]
@dhaval24 Any chance a new release can be created that includes this fix? (or, when will 10.0.11 be released?)

**Comment by dhaval24** [comment]
@HeikoH the next release is tentatively scheduled for the end of November.
